test/common/test/api_utils_test.py

```diff
@@ -0,0 +1,203 @@
+"""
+Copyright 2016, EMC, Inc.
+
+Author(s):
+"""
+import sys
+import subprocess
+
+# set path to common libraries
+sys.path.append(subprocess.check_output("git rev-parse --show-toplevel", shell=True).rstrip("\n") + "/test")
+
+import unittest
+
+#CIT
+from modules.logger import Log
+from on_http_api2_0 import ApiApi as Api
+from config.api1_1_config import config as config_old   #TODO remove when 2.0 worklfow API is implemented
+from config.api2_0_config import config
+from json import loads
+
+from common.api_utils import api_node_select_from_config, api_node_select, api_validate_node_pollers
+
+LOG = Log(__name__)
+
+from nose.plugins.attrib import attr
+@attr(all=True, regression=True, smoke=True)
+
+class NodesTests(unittest.TestCase):
+
+    def setUp(self):
```

Comment:
Is this code a converted CIT script, or is it meant to test api_utils? It feels a bit like the former with the use of CIT things, but, well the name :). 

I find it very hard to tell what each one is _doing_ in any case. "test_all_list" can mean just about anything..
You can use a one-line doc-string to give more detailed info like (for test_all_list):
    "test performing api_utils.nodes_get_all results in same list shows.... what IS this showing?"